src/app/stack/stack.tpl.html

```diff
@@ -22,7 +22,7 @@
                 <i class="fa fa-users fa-4x"></i>
               </div>
               <div class="details">
-                <span class="title">Users</span>
+                <span class="title">{{'UserRatio' | translate}}</span>
```

Comment:
I set this `Users` to `UserRatio`.Because Users have two meaning `count` and `ratio`. In `Sessions Dashboard`  is count.In `Other Dashboard` is ratio.